examples/vision/attention_mil_classification.py

```diff
@@ -0,0 +1,584 @@
+"""
+Title: Classification using Attention-based Deep Multiple Instance Learning (MIL).
+Author: [Mohamad Jaber](https://www.linkedin.com/in/mohamadjaber1/)
+Date created: 2021/08/16
+Last modified: 2021/10/05
+Description: MIL approach to classify bags of instances and get their individual instance score.
+"""
+"""
+## Multiple instance learning
```

Comment:
Please revise the intro. It should be structured as follows, schematically:

```
## Introduction

- What is Multiple Instance Learning (MIL)?
    - Motivation / goals
    - How it works / implementation
    - References
- What you will learn in this example
```

Reply:
Done exactly as the above structure.
I also reduced some text and tried to make it more to the point. Didn't want to shorten it much as some info might be essential for others to help them understand the example.